packages/calcite-components/src/components/carousel-item/carousel-item.tsx

```diff
@@ -0,0 +1,79 @@
+import { Component, Element, h, Host, Prop, VNode } from "@stencil/core";
+import {
+  LoadableComponent,
+  setComponentLoaded,
+  setUpLoadableComponent,
+} from "../../utils/loadable";
+import { guid } from "../../utils/guid";
+import { CSS } from "./resources";
+
+/**
+ * @slot - A slot for adding content.
+ */
+@Component({
+  tag: "calcite-carousel-item",
+  styleUrl: "carousel-item.scss",
+  shadow: true,
+})
+export class CarouselItem implements LoadableComponent {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * When `true`, the component is selected.
+   */
+  @Prop({ reflect: true }) selected = false;
+
+  /**
+   * Accessible name for the component.
+   */
+  @Prop() label!: string;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Private Properties
+  //
+  // --------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteCarouselItemElement;
+
+  private guid = `calcite-carousel-item-${guid()}`;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Lifecycle
+  //
+  // --------------------------------------------------------------------------
+
+  componentDidLoad(): void {
+    setComponentLoaded(this);
+  }
+
+  async componentWillLoad(): Promise<void> {
+    setUpLoadableComponent(this);
+  }
+
+  // --------------------------------------------------------------------------
+  //
+  //  Render Methods
+  //
+  // --------------------------------------------------------------------------
+
+  render(): VNode {
+    const id = this.el.id || this.guid;
+    return (
+      <Host id={id}>
+        <div
```

Comment:
Should this use a `<section>` element? https://www.w3.org/WAI/ARIA/apg/patterns/carousel/examples/carousel-1-prev-next/

I think it would be more semantic if it did.

Reply:
Yeah, using a section element as the correct container would be nice.

Reply:
Yeah I would follow the examples and just have a section at the carousel level.